cmd/daemon_logger/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+)
+
+var (
+	hostport  = flag.String("hostport", "", "Hostname and port of destination.")
```

Comment:
Oh yeah - I remember I asked about this - can we use the veneur config file for this? Would probably make invocation of this tool much easier (and allow things like reporting errors to sentry further down the line).